snowdrop-bot/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Introduction
+
+Ansible playbook to deploy the `snowdrop-bot` into k8s using [quay.io](https://quay.io/) registry.
+
+# Playbook
+
+This project consists of 2 playbooks:
+* [build](#build): builds the project
+* [deploy](#deploy): deploys the application to [quay.io](https://quay.io/) and restarts the k8s pod
```

Comment:
We dont deploy the application but we push the image to the quay registry. Can you then rename `deploy` to `push` and do the same within the role/tasks ?